README_PODMAN.md

```diff
@@ -0,0 +1,44 @@
+# Using the dnsname plugin with Podman
+
+The purpose of the *dnsname* plugin is to allow containers to resolve each other by name.  The plugin adds each
+container's name to an instance of a dnsmasq server.  The plugin is enabled through adding it to a network's
+CNI configuration.  The containers will only be able to resolve each other if they are on the same CNI network.
+
+**Note**: This only works with non-rootless containers.
+
+This tutorial assumes you already have Podman, containernetwork-plugins, and a golang development environment installed.
+
+# Install dnsmasq
+
+Using your package manager, install the *dnsmasq* package.  For Fedora, this would be:
+`sudo dnf install dnsmasq`
+
+
+## Build and install
+
+1. using git, clone the *github.com/containers/dnsname* repository.
+2. make install PREFIX=/usr -- this will install the dnsname plugin into /usr/libexec/cni where your CNI plugins
+should already exist.
+
+## Configure a CNI network for Podman
+
+1. Create a new network using `podman network create`.  For example, `podman network create foobar` will suffice.
+2. Using your favorite editor, edit `/etc/cni/net.d/foobar.conflist` and add the following with the plugins stanza:
+```
+      {
+     "type": "dnsname",
+         "domainName": "podman.io"
+      }
+
+```
+The following example [configuration file](example/cni-podman1.conflist) shows a usable example for Podman.
+
+# Example: container name resolution
+
+1. sudo podman run -dt --name web --network foobar quay.io/libpod/alpine_nginx:latest
+    5139d65d22135e9ecab511559d863754550894a32285befd94dab231017048c2
+
+    Note: we use the --network foobar here. Also, in this test image, the nginx server will respond with
```

Comment:
How do I get this to happen on the `default` network?  Or how do I make `foobar` my default network?